site/src/components/AuditLogRow/AuditLogRow.tsx

```diff
@@ -16,7 +16,28 @@ import userAgentParser from "ua-parser-js"
 import { combineClasses } from "util/combineClasses"
 import { AuditLogDiff } from "./AuditLogDiff"
 
-const readableActionMessage = (auditLog: AuditLog) => {
+// the BE returns additional_field as a string, since it's stored as JSON but
+// technically, it's a map, so we adjust the type here.
+type ExtendedAuditLog = Omit<AuditLog, "additional_fields"> & {
+  additional_fields: Record<string, string>
+}
+
+const readableActionMessage = (auditLog: ExtendedAuditLog) => {
+  // workspace builds audit logs don't have targets; therefore format them differently
+  if (auditLog.resource_type === "workspace_build") {
+    // remove the "{target}" identifier in the string description as we don't use it
+    const amendedDescription = auditLog.description.substring(
+      0,
+      auditLog.description.lastIndexOf(" "),
+    )
+    return amendedDescription
+      .replace("{user}", `<strong>${auditLog.user?.username}</strong>`)
+      .replace(
+        auditLog.additional_fields.workspaceName,
+        `<strong>${auditLog.additional_fields.workspaceName}</strong>`,
+      )
```

Comment:
I think the `replace` on the name here will not match then name since it was stripped out in `amendedDescription`

If workspace name is hello:
`"admin stopped workspace build for hello"` > `"admin stopped workspace build for".replace("hello", "<strong>hello</strong>")` == admin stopped workspace build for
If workspace name is workspace:
`"admin stopped workspace build for workspace"` > `"admin stopped workspace build for".replace("workspace", "<strong>workspace</strong>")` == admin stopped **workspace** build for

If we make the `{target}` change in my other comment we could do the replacement with something like:

```typescript
const target = auditLog.resource_target.trim()
if (auditLog.resource_type === "workspace_build") {
  target = auditLog.additional_fields.workspaceName
}
return auditLog.description
    .replace("{user}", `<strong>${auditLog.user?.username.trim()}</strong>`)
    .replace("{target}", `<strong>${target}</strong>`)
```

Reply:
Ooof, just realized this had the exact same issue you already called out previously - sorry to make you type it out again! I guess I didn't have my coffee this morning.

Reply:
> wrote tests in penance

:joy: :joy: :joy: